cypress/e2e/spotlight/spotlight.spec.ts

```diff
@@ -360,15 +369,37 @@ describe("Spotlight", () => {
         cy.openSpotlightDialog().within(() => {
             cy.spotlightFilter(Filter.People);
             cy.spotlightSearch().clear().type(bot2.getUserId());
+            cy.wait(1000); // wait for the dialog code to settle
             cy.spotlightResults().should("have.length", 2);
             cy.spotlightResults().eq(0).should("contain", `${bot1Name} and ${bot2Name}`);
         });
     });
 
+    // Test against https://github.com/vector-im/element-web/issues/22851
+    it("should not regress by showing the same person result multiple times", () => {
```

Comment:
IMO it's not necessary for the names of tests to say that they're regression tests, especially as that's already heavily implied by the comments

```suggestion
    it("should show each person result only once", () => {
```

Reply:
fair - I wasn't planning on using the word "regress" 400 times when I started writing this, but here we are 😅